docs/sources/developers/http_api/team_sync.md

```diff
@@ -1,8 +1,8 @@
 ---
 aliases:
-  - ../../http_api/external_group_sync/
```

Comment:
nit: should you not leave `- ../../http_api/external_group_sync/` in here as well as an alias for the old link?

Or am I misunderstanding how aliases work?

Reply:
Could not find this in the writing guide, @chri2547 would you advice?

Reply:
@jdbaldry thanks for the feedback. Since one backport is already merged, I think the optimal thing would be to send a new PR to add back the alias, instead of reverting, this will keep history cleaner. Will send a PR ASAP.

Reply:
Sure thing, I'll remove my approval from Chris' revert PR. https://github.com/grafana/grafana/pull/69404

Reply:
https://github.com/grafana/grafana/pull/69434 here is the new PR